CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ _None_
 - Add "Mobile Secrets" to `configure_update` current branch message to clarify which repo it's referring to. [#455]
 - `buildkite_trigger_build` now prints the web URL of the newly scheduled build, to allow you to easily open it via cmd-click. [#460]
 - Add the branch information to the 'This is not a release branch' error that's thrown from complete code freeze lane. [#461]
+- Adds `ignore_pipeline_branch_filters=true` parameter to the API call triggering a Buildkite build [#468]
```

Comment:
I don't think this needs to be in the breaking change section because it doesn't break any existing triggers. It only adds the possibility to trigger builds for projects where the GitHub activity is turned off.

Admittedly, `release-toolkit-demo` is a special case, as we have the GitHub activity turned on for all our other projects (at least as far as I know). However, I think it's best to address it now that we know about it and avoid any further debugging in case it happens in another special case.